$NetBSD: patch-CMakeLists.txt,v 1.5 2025/10/13 02:46:50 mrg Exp $

Newer boost obsoleted "boost_system", so look for a different component.


--- CMakeLists.txt.orig	2025-07-12 12:30:12.000000000 -0700
+++ CMakeLists.txt	2025-10-12 17:43:55.295154207 -0700
@@ -813,7 +813,7 @@ if (MINGW AND CMAKE_CXX_COMPILER_ID STRE
     set(Boost_COMPILER "mgw${KRITA_boost_COMPILER_VERSION}")
 endif()
 
-find_package(Boost 1.65 REQUIRED COMPONENTS system)
+find_package(Boost 1.65 REQUIRED COMPONENTS process)
 set_package_properties(Boost PROPERTIES
     DESCRIPTION "Boost provides free peer-reviewed portable C++ source libraries."
     URL "https://www.boost.org"
